scripts/create-agent.ts

```diff
@@ -388,7 +388,17 @@ AGENT_ENVIRONMENT=production`;
       ];
 
       for (const commandObj of setupCommands) {
-        await this.executeSSMCommand(instanceDetails.instanceId, commandObj);
+        try {
```

Comment:
This try catch should likely be a part of the execute ssm command helper directly